### ISSUE TITLE
Remove Table Styles from edd.css

### ISSUE THREAD

**Issue by spencerfinnell** [easydigitaldownloads/Easy-Digital-Downloads#3892]
I can't really think of any theme (or at least there shouldn't be) that doesn't provide their own table styles. EDD provides just a few, but the places it does usually override the theme's defaults because they are styled via an ID.

Not sure applying things like `padding` really make sense in the plugin. Certain items like `vertical-align` or `text-align` __maybe__, but adding borders and padding seems like a bit much to me.

A standard table like the in the Purchase History page shouldn't need any extra styles. It's as basic as a table can get. The theme should have styles for that already.

**Comment by pippinsplugins** [comment]
I disagree, simply based on my experience with EDD, RCP and AffiliateWP.

None of the plugins used to supply any CSS for tables at all, for exactly the reasons you gave, @spencerfinnell.

The problem we ran into, however, is that there is a hugely unfortunate number of themes out there that do not properly style tables, which results in really crappy display.

We found the best option was to provide just very, very basic styles to ensure that table data is displayed in a presentable manner. With some basic styles, it's still easy for themes to override them but it ensures that things don't look like junk.

It's the same reason we supply basic styles for checkout.

**Comment by spencerfinnell** [comment]
Reasonable. 

What about a documentation article that provides some boilerplate CSS? Or move the styles to a class so the ID style doesn't override more generic CSS?

**Comment by pippinsplugins** [comment]
I would love to see that.

Perhaps also basing all of our styles off of classes instead of IDs. Would that help @spencerfinnell ?

**Comment by spencerfinnell** [comment]
Yes removing all styles from IDs would definitely help. Easy backwards compat because the IDs can stay for people styling off of them.

**Comment by kemie** [comment]
Reviving an old thread here. I'm having a really hard time overriding edd.min.css because it's CSS with high specificity,and opinionated graphic styles. There's no need to specify  border radius and border color on the checkout button, it should inherit what is in my theme.
